user-service/src/main/resources/application.yaml

```
@@ -1,8 +1,17 @@
   application:
     name: user-service
+  datasource:
+    url: ${secrets.datasource.url}
+    username: ${secrets.datasource.username}
+    password: ${secrets.datasource.password}
+    driver-class-name: org.postgresql.Driver
+  config:
+    import: classpath:secrets.yml
+
 server:
     port: 8083
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 4

(indentation)